### ISSUE TITLE
Mac Mini Late 2012 High Sierra

### ISSUE THREAD

**Issue by Jensilein** [Floris497/mac-pixel-clock-patch-V2#244]
Hi Floris497, 

can you please add the following md5:

---- BEGINNING MD5 HASH SUMS ---- version: 10.12.6 build:16G1114

     otool IOKit: 449aceaeea2662eeb2625fdf9473cd76
 otool IOKit.bak: 991df1fc5d2538f50a7e58e55f3a8fba
           IOKit: adf5a45c965551bc6ca2998845a86f2a
       IOKit.bak: 4807723521d524e54e4c3583cc49fec6

---- ENDING MD5 HASH SUMS -------

Thanks.

**Comment by Floris497** [comment]
Hm.. you are not running High Sierra.. I'll check it out tomorrow

**Comment by Jensilein** [comment]
Hi Floris496,
did you already have the chance to check it out? Feedback would be very appreciated.
Thx

**Comment by Floris497** [comment]
Force it like described in the readme :) I'm sorry I've no time really to update the script

**Comment by Floris497** [comment]
`CoreDisplay-patcher.command patch v3` (for 10.12.6 you would most likely need v3)

**Comment by Jensilein** [comment]
Thanks for your reply. To prevent misunderstanding: on my Mac Mini Late 2012 currently OSX Sierra is running with your Pixel Clock patch successfully applied. Now I want to upgrade to OSX High Sierra and I‘m wondering whether I can apply the Pixel Clock Patch too... If I understood you correctly I have to apply patch v3 and that will work with High Sierra?

**Comment by Floris497** [comment]
Hm, but if you are on HS it should patch automatically with the correct version :)

**Comment by Jensilein** [comment]
No, at the moment I‘m not on HighSierra but I want to upgrade now. Can I apply the Pixel Clock Patch after the upgrade on HighSierra upgrade and High Resolution output on HDMI and DisplayPort will still work as fine as under Sierra?

**Comment by Floris497** [comment]
As far as I know it works fine on HS :)
